lib/services/runtime/handlers/interaction.ts

```diff
@@ -28,7 +28,9 @@ export const InteractionHandler: HandlerFactory<
 > = (utils) => ({
   canHandle: (node) => !!node.interactions,
   handle: (node, runtime, variables) => {
-    if (runtime.getAction() === Action.RUNNING) {
+    const isStartingFromButtonsStep = runtime.getAction() === Action.REQUEST && !runtime.getRequest();
+
+    if (runtime.getAction() === Action.RUNNING || isStartingFromButtonsStep) {
```

Comment:
Could just grab the action once and avoid updating tests.

```suggestion
    const runtimeAction = runtime.getAction();
    const isStartingFromButtonsStep = runtimeAction === Action.REQUEST && !runtime.getRequest();

    if (runtimeAction === Action.RUNNING || isStartingFromButtonsStep) {
```